cleanlab_studio/studio/enrichment.py

```diff
@@ -49,6 +50,13 @@ def _response_timestamp_to_datetime(timestamp_string: str) -> datetime:
     return datetime.strptime(timestamp_string, response_timestamp_format_str)
 
 
+@lru_cache(maxsize=None)
+def _get_run_online():
+    from cleanlab_studio.utils.data_enrichment.enrich import run_online
+
+    return run_online
+
```

Comment:
get rid of this and do a standard import, unsure why you are using some an odd approach